src/callbacks.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 /* When returning promises becomes the default in some future version, this
  * file will start modifying the exports as promises.js does today.
- * Please note that the file imported below is only generated after running 
+ * Please note that the file imported below is only generated after running
  * the build task. */
-module.exports = require('./browser/static/ably-node');
+module.exports = require('../browser/static/ably-node');
```

Comment:
I'm guessing this import won't work anymore since ably-node has moved to the `dist` folder

Reply:
Fixed [here](https://github.com/ably/ably-js/pull/982/commits/f042f1cb73ddd4c122b807c8f712f875cdd26277)